value.go

```diff
@@ -69,6 +69,8 @@ var errDeleteVlogFile = errors.New("Delete vlog file")
 
 type logEntry func(e Entry, vp valuePointer) error
 
+type ReqValSize []int64
```

Comment:
Do we need a separate type?

Reply:
It was looking bit noisy to add int64 everywhere i.e. channels, req tracker etc. I did this so as to add context to the variable. Any thoughts about this?